results/cmd/api/results.sql

```diff
@@ -1,2 +1,9 @@
+<<<<<<< HEAD
```

Comment:
Looks like we need to resolve a diff here.

Reply:
Sure.